redfish-core/include/utils/error_log_utils.hpp

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
```

Comment:
include what you use

Reply:
Can you run https://include-what-you-use.org/ ?

Reply:
Thank you for your valid comments. We have created the JIRA ticket https://jsw.ibm.com/browse/PFEBMC-620 to address this comment.
The scope of this task was to cherry pick the working HWIsolation commits from 1030. And as the number of commits are more, it is difficult to refactor them to align with upstream as part of this PR. So we will address all them as a separate story.